.github/workflows/linux.yml

```diff
@@ -530,18 +530,14 @@ jobs:
       runner: 'ubuntu-20.04-16-cores'
       model_scope: 'nightly_hf'
 
-  # TODO: Switch back to self-hosted runners
-  # container:
-  #   image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
-  #   volumes:
-  #     - /mount:/mount
   PyTorch_Models_Tests:
     name: PyTorch Models tests
     if: fromJSON(needs.smart_ci.outputs.affected_components).PyTorch_FE.test
     needs: [ Build, Smart_CI, Openvino_tokenizers ]
     uses: ./.github/workflows/job_pytorch_models_tests.yml
     with:
-      runner: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-16-cores' || 'ubuntu-20.04-8-cores' }}
+      runner: ${{ github.event_name == 'schedule' && 'aks-linux-16-cores-32gb' || 'aks-linux-8-cores-16gb' }}
+      container: '{"image": "openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04", "volumes": ["/mount:/mount"]}'
       event: ${{ github.event_name }}
 
```

Comment:
you also need to do with this: https://github.com/openvinotoolkit/openvino/blob/master/tests/model_hub_tests/models_hub_common/constants.py#L30
set USE_SYSTEM_CACHE and also set HUGGINGFACE_HUB_CACHE.
Besides, I am not sure if need to have common variable for both PyTorch and TensorFlow HuggingFace models. I think we need to split and rename current variable for it.
@mvafin, please comment

Reply:
Hi @ababushk, we decided do not split cached TF and PR models now. That is because we have limitation in tf_sentence_transformers package that does not allow to set cache dir via method option for model loading. It requires to use common HF env variable.
So please proceed with your PR.